calculator-domain/src/main/java/camp/nextstep/edu/calculator/domain/RecordRepository.kt

```diff
@@ -0,0 +1,13 @@
+package camp.nextstep.edu.calculator.domain
+
+
+import camp.nextstep.edu.calculator.domain.model.Record
+
+interface RecordRepository {
+
+    suspend fun insertRecord(record: Record)
+
+    suspend fun deleteRecord(record: Record)
+
+    suspend fun getRecord(): List<Record>
```

Comment:
```suggestion
    suspend fun getRecords(): List<Record>
```

리스트와 같이 여러개를 나타내는 경우 복수형을 사용해주는 것이 좋습니다.

Reply:
오넵! 자꾸 까먹네요~ 감사합니당!